client-metrics-reporter/src/test/java/io/strimzi/kafka/metrics/prometheus/MetricsUtils.java

```diff
@@ -155,15 +169,16 @@ public static GenericContainer<?> clientContainer(Map<String, String> env, int p
     }
 
     /**
-     * Start a connector
+     * Start a connector and ensure its tasks are running
      * @param connect the Connect cluster
      * @param name the name of the connector
      * @param config the connector configuration
+     * @param expectedTasks the number of tasks
      */
-    public static void startConnector(StrimziConnectCluster connect, String name, String config) {
+    public static void startConnector(StrimziConnectCluster connect, String name, String config, int expectedTasks) {
         assertTimeoutPreemptively(TIMEOUT, () -> {
+            HttpClient httpClient = HttpClient.newHttpClient();
             while (true) {
```

Comment:
Can we add in a liner comment what this loop does e.g., `// Retry connector creation`?